data/helpers.d/filesystem

```diff
@@ -46,6 +47,7 @@ ynh_backup() {
     local SRC_PATH="$1"
     local DEST_PATH="${2:-}"
     local IS_BIG="${3:-0}"
+    local NOT_MANDATORY="${4:-0}"
```

Comment:
If there isn't a 4th argument, could i suggest to put 1 by default if $SRC_PATH contains "fail2ban" ? (same in restore)

Reply:
There's already this PR  (https://github.com/YunoHost/yunohost/pull/558) for fail2ban.

Reply:
ok